Paolo theme protocol (#8)
RISC-OS-Community/DME-Core: doc/README.md

# DME-Core documentation
This is the "ConfigurationObject" for the entire DME.

> It reads all the configuration files, the Operating System version and the Window version and apply a desired configuration accordingly.

Reviewer: ```suggestion
It reads all the configuration files, the Operating System version and the Window version and applies a desired configuration accordingly.
```

## Requirements
[Detailed requirement list here](00DMERequirements.md)

## Design
[DME Design is available here](01DMEDesign.md)

## DME Config File Protocol
[DME Configuration File Protocol details available here](02DMEConfigFileP.txt)

## DME Theme Protocol

> The DME Theme Protocol is inspired by Rich Goodwin and Chir (Little Yellow Moon) works.

Reviewer: ```suggestion
The DME Theme Protocol is inspired by Rich Goodwin's and Chir's (Little Yellow Moon) works.
```
 Altereatively `The DME Theme Protocol is inspired by the works of Rich Goodwin's and Chir's (Little Yellow Moon).`

> It foundamentally is a text file that describes all the options and details to apply a theme correctly (where correctly means "as the original theme author meant it to be applied")
> 
> This type of feature does not exists in the bare RISC OS and so must be added in order to make DME be able to deal with complex themes that may require more than just reload the tool sprites.

Reviewer: ```suggestion
It is fundamentally a text file that describes all the options and details to apply a theme correctly (where correctly means "as the original theme author meant it to be applied")

This type of feature does not exists in the bare RISC OS and so must be added in order to make DME be able to deal with complex themes that may require more than to just reload the tool sprites.
```
